src/game_interpreter.cpp

```diff
@@ -176,6 +176,9 @@ void Game_Interpreter::Update() {
 			if (Main_Data::game_player->IsBoardingOrUnboarding())
 				break;
 
+			if (Main_Data::game_player->InVehicle() && Main_Data::game_player->GetVehicle()->IsAscendingOrDescending())
+				break;
```

Comment:
Another timing issue with vehicles, we had an almost similiar one in Witch's Heart before: https://github.com/EasyRPG/Player/commit/9659baeac5e4c6be9046fac9597f4b0aa83fc0d2